src/Rector/v8/v3/RefactorQueryViewTableWrapRector.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Ssch\TYPO3Rector\Rector\v8\v3;
+
+use PhpParser\Node;
+use PhpParser\Node\Arg;
+use PhpParser\Node\Expr\MethodCall;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+use TYPO3\CMS\Core\Database\QueryView;
+
+/**
+ * @see https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/8.3/Deprecation-77557-MethodQueryView-tableWrap.html
+ */
+final class RefactorQueryViewTableWrapRector extends AbstractRector
+{
+    /**
+     * @return string[]
+     */
+    public function getNodeTypes(): array
+    {
+        return [MethodCall::class];
+    }
+
+    /**
+     * @param MethodCall $node
+     */
+    public function refactor(Node $node): ?Node
+    {
+        if (! $this->isMethodStaticCallOrClassMethodObjectType($node, QueryView::class)) {
+            return null;
+        }
+        if (! $this->isName($node->name, 'tableWrap')) {
+            return null;
+        }
+
+        ///** @var Arg[] $args */
+        //$args = $node->args;
+        //$firstArgument = array_shift($args);
```

Comment:
Again having hard time figuring out how to wrap the `$firstArgument` with `<pre>` similar to in https://github.com/sabbelasichon/typo3-rector/pull/1637

Reply:
What you need in this case it to return a node of type Concat
return new Concat(new Concat(new String_('<pre>'), $firstArgument->value), new String_('</pre>'));

Reply:
This look really ugly, but it is the way how it works. Getting nodes and writing nodes all the way down. You have to get used to it.

Reply:
It's a little better without the full namespaces :)